### PR TITLE
add all necessary data to admin

### PR DESCRIPTION
**Pull request by annashamray**
issue - https://github.com/open-zaak/open-zaak/issues/13

### CODE REVIEW

File: src/openzaak/components/besluiten/admin.py [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+)
 
 from .models import Besluit, BesluitInformatieObject
 
 
-class BesluitInformatieObjectInline(AuditTrailInlineAdminMixin, admin.TabularInline):
-    model = BesluitInformatieObject
-    extra = 0
-    readonly_fields = ("uuid",)
-    raw_id_fields = ("_informatieobject",)
```
Comment (stevenbal) [inline src/openzaak/components/besluiten/admin.py]: Now `_informatieobject` is no longer a `raw_id_field` right?

Comment (annashamray) [reply src/openzaak/components/besluiten/admin.py]: `BesluitInformatieObjectInline` has now `EditInlineAdminMixin`, and it seems that now `raw_id_fields` has no effect on the representation of inline object.

File: src/openzaak/components/documenten/admin.py [hunk 2]
```diff
@@ -18,13 +19,45 @@
...
+    ordering = ("startdatum", "informatieobject")
+    raw_id_fields = ("informatieobject",)
+    viewset = viewsets.GebruiksrechtenViewSet
+
+
+@admin.register(ObjectInformatieObject)
+class ObjectInformatieObjectAdmin(
+    AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmin
+):
+    list_display = ("informatieobject", "object_type", "zaak", "besluit")
```
[the remark this answers is not in the document]
Comment (annashamray) [reply src/openzaak/components/documenten/admin.py]: thanks! I haven't checked it after rebase. Added custom `object` field to OIO admin list view

**Comment by annashamray** [comment]
@sergei-maertens thank you for all improvements

**Pull request merged**